compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/DocumentationAnalyzer.java

```diff
@@ -323,11 +323,17 @@ private BSymbol resolveFullyQualifiedSymbol(DiagnosticPos pos, SymbolEnv env, St
 
         // If there is no type in the reference we need to search in the package level and the current scope only.
         if (typeName == Names.EMPTY) {
-            return symResolver.lookupSymbolInPackage(pos, env, pkgName, identifierName, tag);
+            if ((tag & SymTag.IMPORT) == SymTag.IMPORT) {
+                return symResolver.lookupPrefixSpaceSymbolInPackage(pos, env, pkgName, identifierName);
+            } else if ((tag & SymTag.ANNOTATION) == SymTag.ANNOTATION) {
+                return symResolver.lookupAnnotationSpaceSymbolInPackage(pos, env, pkgName, identifierName);
```

Comment:
I think this validation is not needed. `type` defined only in the main symbol space.

Reply:
But according to [this](https://github.com/ballerina-platform/ballerina-lang/blob/f5db13f2fbec192b2906644ff517d4abc0e5bb3f/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/DocumentationAnalyzer.java#L256), SymTag can be ANNOTATION, or IMPORT. Therefore don't we need to validate the symbol.